dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -748,13 +748,46 @@ const borderStandfirstLink = (format: ArticleFormat): string => {
 };
 
 const borderHeadline = (format: ArticleFormat): string => {
-	if (format.design === ArticleDesign.LiveBlog) return '#9F2423';
+	if (format.design === ArticleDesign.LiveBlog) {
+		switch (format.theme) {
+			case ArticlePillar.Culture:
+				return '#716960';
+			case ArticlePillar.Opinion:
+				return '#ae4017';
+			case ArticlePillar.Lifestyle:
+				return '#713b68';
+			case ArticlePillar.Sport:
+				return '#3e657b';
+			case ArticleSpecial.SpecialReport:
+				return specialReport[200];
+			case ArticleSpecial.Labs:
+				return labs[200];
+			case ArticlePillar.News:
+				return '#914b4a';
```

Comment:
Where do these colours come from? Is `decidePalette` the right place to introduce new colour hexes?

Reply:
I would be cautious adding new colours as it makes colour contrast harder to guarantee at scale. If these borders can be considered decorative, it might not matter. But deciding what is decorative is a non-trivial exercise, and if this question came up every time you make a colour change, it would slow down development.

Reply:
What's the alternative?

These particular colours are tricky to get right as there is this strange effect where the surrounding colour changes how you perceive them so I suspect they are only going to exist in this one place. 

All that said, I'm very happy to adopt a different strategy here but would prefer a staged approach that doesn't block this PR 🙏 as this is going to help us with the liveblog launch (which is coming soon!).

Reply:
One alternative could be to define a transparent white or transparent black colour, which would introduce less changes, like so:

```ts
		switch (format.theme) {
			case ArticlePillar.Culture:
			case ArticlePillar.Opinion:
			case ArticlePillar.Lifestyle:
				return 'rgba(255,255,255, 0.5)';
			case ArticlePillar.Sport:
			case ArticlePillar.News:
				return 'rgba(0,0,0, 0.5)';
			case ArticleSpecial.SpecialReport:
				return specialReport[200];
			case ArticleSpecial.Labs:
				return labs[200];
```

But I don’t see this as a blocker for this PR.

Reply:
> transparent

Clever! I guess a custom colour is more, well, customisable but maybe we can get what we need using RGB @HarryFischer what do you think?

| Custom hex | RGB 0.5 | RGB 0.3 |
|------------|---------|---------|
|       <img width="122" alt="Screenshot 2022-03-17 at 11 06 25" src="https://user-images.githubusercontent.com/1336821/158796685-9a6e523e-1101-41b6-82a1-918caed176ce.png">    |   <img width="122" alt="Screenshot 2022-03-17 at 11 06 48" src="https://user-images.githubusercontent.com/1336821/158796746-608c02bd-8e17-4f42-a1c1-190f5fb7cd9c.png">      |    <img width="122" alt="Screenshot 2022-03-17 at 11 07 22" src="https://user-images.githubusercontent.com/1336821/158796775-ca801f9a-eda0-4643-9f9c-0ad8f07da625.png">     |

Reply:
@oliverlloyd RGB 0.3 is best for me

Reply:
I refactored everything to use RGB 👍 

@HarryFischer - I went with 0.2 as it seemed to look best over all the different themes but can change it back if you prefer?

### `0.2`
![image](https://user-images.githubusercontent.com/1336821/158806118-32738856-7fda-4adb-a03a-9e0384832b24.png)